modules/weboramaRtdProvider.md

```diff
@@ -546,15 +546,15 @@ pbjs.que.push(function () {
                     },
                     weboUserDataConf: {
                         accountId: 12345,           // recommended
-                        setPrebidTargeting: ['adUnitCode1',...], // set target only on certain adunits 
+                        setPrebidTargeting: ['adUnitCode1',...], // set target only on certain adunits
                         sendToBidders: { // send to only some bidders and adunits
-                            'appnexus': true,               // all adunits for appnexus 
+                            'appnexus': true,               // all adunits for appnexus
                             'pubmatic': ['adUnitCode1',...] // some adunits for pubmatic
                             // other bidders will be ignored
                         },
                         defaultProfile: {           // optional
-                            webo_cs: ['Red'],
-                            webo_audiences: ['bam']
+                            webo_cs: ['Red'],       // using label
+                            webo_audiences: [12345] // using id
```

Comment:
one example of test changed, we verify if a numeric audience still works. most of the tests use a string on webo_cs (stands for custom segment) as string and audience as integer but in one test we use the opposite just in case.